src/tiles/grid/item-tile.ts

```diff
@@ -76,12 +73,17 @@ export class ItemTile extends LitElement {
             <span>${itemCreator}</span>
           </div>
         </div>
+
+        <div class="hr"></div>
```

Comment:
I think my other comment got lost in the rebase, but do you need a separate div for this horizontal line or can you set a `border-top` on the `#item-stats-container`?